.github/workflows/tests.yml

```diff
@@ -212,6 +215,13 @@ jobs:
 
       - uses: codecov/codecov-action@e156083f13aff6830c92fc5faa23505779fbf649 # v1.2.1
 
+      - name: Cache parallel tests log
+        uses: actions/cache@v1
+        with:
+          path: tests/parallel_runtime_rspec.log
+          key: ${{ runner.os }}-parallel_runtime_rspec.log
+          restore-keys: ${{ runner.os }}-parallel_runtime_rspec.log
```

Comment:
This is the point where it restores the log. I think you want it before `brew tests`.

(The actual caching is automatically done at the end of the job)